awx/main/tests/functional/test_licenses.py

```diff
@@ -0,0 +1,135 @@
+
+import glob
+import json
+import os
+import urlparse
+
+from django.conf import settings
+
+
+def test_python_and_js_licenses():
+
+    def index_licenses(path):
+        # Check for GPL (forbidden) and LGPL (need to ship source)
+        # This is not meant to be an exhaustive check.
+        def check_license(file):
+            with open(file) as f:
```

Comment:
This is a bit nitpick, but using `file` as a variable name is frowned upon, because it's a built-in.